tsl/src/chunk.c

```diff
@@ -175,92 +175,3 @@ chunk_set_default_data_node(PG_FUNCTION_ARGS)
 
 	PG_RETURN_BOOL(chunk_set_foreign_server(chunk, server));
 }
-
-/* Should match definition in ddl_api.sql */
```

Comment:
How is drop_chunk executed on data node? I don't see references to this code, but do we need it for that case?

Reply:
Just talked to @mkindahl and we agreed that we'd like to keep this function and submit a PR to reenable use of it.

Reply:
ok i'll not remove it in this PR then